commons-business/src/main/java/com/appjars/saturn/service/CreationService.java

```diff
@@ -20,15 +20,21 @@
 package com.appjars.saturn.service;
 
 /**
- * A special kind of service that allows entities creation
- * 
- * @author mlopez
+ * Specifies the contract of a generic service that is responsible for saving new entities to a database.
```

Comment:
IMO, `for saving new entities to the* database.` sounds better than `for saving new entities to a database.`